lib/chat_api_web/controllers/setup_status_controller.ex

```diff
@@ -0,0 +1,73 @@
+defmodule ChatApiWeb.SetupStatusController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.{
+    Accounts,
+    BrowserSessions,
+    Github,
+    Google,
+    Mattermost,
+    SlackAuthorizations,
+    Twilio,
+    Users
+  }
+
+  action_fallback(ChatApiWeb.FallbackController)
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, _params) do
+    with current_user <- Pow.Plug.current_user(conn),
+         %{account_id: account_id} <- current_user do
+      account = Accounts.get_account!(account_id)
+
+      json(conn, %{
+        configured_profile: configured_profile?(current_user.id),
+        configured_storytime: configured_storytime?(account_id),
+        has_integrations: has_integrations?(account_id),
+        installed_chat_widget: installed_chat_widget?(account),
+        invited_teammates: invited_teammates?(account_id),
+        upgraded_subscription: upgraded_subscription?(account)
```

Comment:
nit - i'm not always good about this myself, but i think it's nice to name boolean fields with `is_`/`has_`/`should_` prefixes to make it really obvious that the fields are boolean, e.g.
```javascript
{
  has_configured_profile: boolean; // or `is_profile_configured`
  has_configured_storytime: boolean; // or `is_storytime_configured`
  has_integrations: boolean;
  is_chat_widget_installed: boolean;
  has_invited_teammates: boolean;
  has_upgraded_subscription: boolean;
};
```
(otherwise i might expect `invited_teammates` to return the list of teammates, or `upgraded_subscription` to return a subscription object, etc.)

Reply:
Yeah, I totally agree actually that boolean fields should always be prefixed with one of those prefixes. I also am not always good about it 😅 I'll update it. Thanks for the suggestion.